.gitignore

```diff
@@ -23,4 +23,8 @@ __pycache__
 # Ignore development npm files
 node_modules
 
+# Ignore goturn model
+goturn.caffemodel
+goturn.prototxt
```

Comment:
AFAIK this model is very large. So placing it in source tree would lead to docker builds slowdown.

What is about `.dockerignore`? (if you want to add it where)
Or remove from both places.